lib/rummager/app.rb

```diff
@@ -73,12 +73,16 @@ def json_only
     halt(400, env['sinatra.error'].message)
   end
 
-  error MetasearchIndex::Deleter::V2::UnknownError, MetasearchIndex::Deleter::V2::MissingArgument,
-    MetasearchIndex::Inserter::V2::UnknownError, MetasearchIndex::Inserter::V2::MissingArgument do
+  error Index::ResponseValidator::ElasticsearchError do
+    # TODO: we want to one to be reported to sentry - check this is the case.
```

Comment:
Still a TODO here

Reply:
it because I'm not sure how to test this. and ensure the error is reported.

Reply:
was going to push it live with the todo while I think about it, try to work out a plan. :(

Maybe integration testing will give me inspiration